### ISSUE TITLE
[feature request] Send MYST to other users from within the app

### ISSUE THREAD

**Issue by gir489** [mysteriumnetwork/mysterium-vpn-desktop#281]
**Is your feature request related to a problem? Please describe.**
When buying MYST, I feel it should be mine to do with as I please. Sending MYST across the network is arduous and requires a great deal of knowledge about the internal workings of MYST and how Ethereum tokens are sent. There should be a UI page under the MYST page (currently 60% of the screen real estate from that page is unused) so when trying to convince friends to try Mysterium, they don't want to give their credit card to you, or send BTC either as the cost to send a small amount of BTC is currently outrageous. I think alleviating this by being able to send them MYST across the network for them to try would convince them to use Mysterium.

**Describe the solution you'd like**
Some kind of UI page to send/receive tokens.

**Describe alternatives you've considered**
Manually setting up an ERC20 wallet with my JSON backup, but the parameters did not correlate to anything in the ERC20 wallet app was expecting.

**Comment by Sermooh** [comment]
I completely agree and I tell them all the time in the discord. Without internal transfers between MystIDs, the popularity of the application will grow very slowly.
In the current situation, I use LTC and DOGE, transferring from the FTX exchange without transfer fees.

**Comment by gir489** [comment]
>I use LTC and DOGE, transferring from the FTX exchange without transfer fees.
Having to rely on shitcoins to propagate your network is bad for business.  It's hard enough to convince people to use BTC, an already widely used standard crypto coin, it's even harder to get them to use the silver of crypto ETH. It would be an exercise in futility to get them to adopt a shitcoin like LTC or even DOGE. But I'm glad I'm not the only one kvetching about this.
